the-new-octal/src/app/service/product.service.ts

```diff
@@ -0,0 +1,39 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { environment } from 'src/environments/environment';
+import { Product } from '../model/product';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProductService {
+
+  apiUrl: string = `${environment.apiUrl}product`;
+  endPoint: string = 'product';
+
+  constructor(
+    private http: HttpClient
+  ) { }
+
+  getAll(): Observable<Product[]> {
+    return this.http.get<Product[]>(this.apiUrl);
```

Comment:
Kimaradt az `entityName` / `endPoint`

Reply:
Igazából nem maradt ki, a 12. sorban:
apiUrl: string = `${environment.apiUrl}product`;
beleégettem. De nem is ez a poén, hanem, hogy emlékem szerint én ezt az egészet átírtam a base.service-re építve, itt meg még eredeti állapotában van...( lehet, hgy kimaradt egy commit?), ezt most megpróbálom visszaírni a base-re.alapozva.